ws-nextjs-app/pages/[service]/send/[id]/UGCPageLayout.tsx

```diff
@@ -23,16 +24,25 @@ const UGCPageLayout = ({ initialScreen = 'form', pageData }: PageProps) => {
       } = {},
     },
   } = useContext(ServiceContext);
-  const { title, description, sections, privacyNotice } = pageData;
+
+  const {
+    title,
+    description,
+    sections,
+    privacyNotice,
+    campaignStatus,
+    closingTime,
+  } = pageData;
 
   const { fields } = sections?.[0] ?? {};
   const sectionTitle = sections?.[0].sectionText?.title ?? '';
+
   return (
     <>
       <Metadata
         title={title}
         lang={lang}
-        description="Test UGC Form"
+        description=""
```

Comment:
Left this blank as the existing UGC forms have no description. Not sure if that was intentional or not as the SEO around these pages is a bit unknown.